src/rv32_template.c

```diff
@@ -57,9 +57,21 @@ RVOP(jalr, {
         rv->X[ir->rd] = pc + 4;
     /* check instruction misaligned */
     RV_EXC_MISALIGN_HANDLER(pc, insn, false, 0);
+    /* lookup branch table */
```

Comment:
Could you consider eliminating redundant code for branch lookups in `jalr`, `c.jr`, and `cjalr`? Macros might be a feasible solution.

Reply:
I eliminate redundant code by macro.